gem/templates/chhaajaa/surveys/surveys_list.html

```diff
@@ -0,0 +1,163 @@
+{% load personalise_extras wagtailimages_tags wagtailcore_tags wagtailsettings_tags static gem_tags %}
+{% get_settings %}
+
+{% block content %}
+{% filter_surveys_by_segments surveys request as filtered_surveys %}
+  {% comment %}
+    {# GER:Is feature supported?
+      <!-- PROFILE JOIN BANNER --> #}
+  {% endcomment %}
+  {% if filtered_surveys %}
+    {% with survey=filtered_surveys|first %}
+      {% with survey=survey.molo_survey_page %}
+          {% if not user.is_authenticated and not request.is_preview and not survey.allow_anonymous_submissions and settings.gem.GemSettings.show_join_banner %}
+            {% if not survey.your_words_competition and not survey.display_survey_directly %}
+            <div class="surveys surveys--profiles-join">
+              <div class="surveys__wrapper">
+                <div class="heading heading heading--secondary">
+                  <h1 class="heading__title">{% trans 'Quiz' %}</h1>
+                </div>
+                <div class="surveys__inner">
+                  <section>
+                    <h1 class="heading__headline">
+                      {% trans "Share your opinions and stories, take polls, win fun prizes." %}
```

Comment:
some of this Springster text will have to be taken out, will it be done so at a later stage?

Reply:
@codiebeulaine 
Updated to generic text.